soccer/src/soccer/strategy/agent/position/offense.hpp

```diff
@@ -14,6 +14,9 @@
 #include "rj_geometry/geometry_conversions.hpp"
 #include "rj_geometry/point.hpp"
 
+#include <rclcpp/rclcpp.hpp>
```

Comment:
try not to update or reformat files that your PR isn't intended to fix. if you notice bugs open a separate one